src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts

```diff
@@ -65,22 +65,22 @@ describe('Detailed view and edit a selected work item - ', () => {
     fakeUserList = [
       {
         attributes: {
-          fullName: 'Harry Potter',
```

Comment:
Does this change make difference or it's just to stay consistent with the mock data for functional test?